app/src/main/java/org/oppia/app/options/StoryTextSizeActivityPresenter.kt

```diff
@@ -0,0 +1,87 @@
+package org.oppia.app.options
+
+import android.content.Intent
+import android.widget.SeekBar
+import androidx.appcompat.app.AppCompatActivity
+import androidx.databinding.DataBindingUtil
+import org.oppia.app.R
+import org.oppia.app.activity.ActivityScope
+import org.oppia.app.databinding.StoryTextSizeActivityBinding
+import javax.inject.Inject
+
+/** The presenter for [StoryTextSizeActivity]. */
+@ActivityScope
+class StoryTextSizeActivityPresenter @Inject constructor(
+  private val activity: AppCompatActivity
+) {
+  private var fontSize: String = STORY_TEXT_SIZE_SMALL
+
+  fun handleOnCreate(prefSummaryValue: String) {
+    val binding =
+      DataBindingUtil.setContentView<StoryTextSizeActivityBinding>(activity, R.layout.story_text_size_activity)
+
+    fontSize = prefSummaryValue
+
+    binding.storyTextSizeToolbar.setNavigationOnClickListener {
+      val intent = Intent()
+      intent.putExtra(MESSAGE_STORY_TEXT_SIZE, fontSize)
+      (activity as StoryTextSizeActivity).setResult(1, intent)
+      activity.finish()
+    }
+
+    when (prefSummaryValue) {
+      STORY_TEXT_SIZE_SMALL -> {
+        binding.storyTextSizeSeekBar.progress = 0
+        binding.previewTextview.textSize = 16f
```

Comment:
The constant values in this file are getting reused and if by mistake we change 16f to 15f in one place and forget about the second place it will bring inconsistency. 

So define these constants at the top like:

`private const val SMALL_TEXT_SIZE_VALUE = 16f`

Reply:
Done thanks.